### DISCUSSION TITLE
How to plot multi-series area chart with different baselines

### DISCUSSION THREAD

**Discussion by gvanhorne** [observablehq/plot#2118] [Q&A] [answered]
Hello!

I'm trying to implement a horizon chart similar to https://observablehq.com/@d3/horizon-chart-ii using Plot. In particular, I want to display negative values as "hanging" from the top of the frame, and positive values as normal. I'm having trouble translating the d3 logic into terms that Plot understands. 

Any advice would be appreciated, thanks!

**Comment by Fil** [comment, marked as answer]
See https://observablehq.observablehq.cloud/pangea/plot/diverging-horizon-chart for an example.

**Reply by gvanhorne** [reply]
This was exactly what I was looking for! Thanks very much 🙌
